constraint/pkg/client/drivers/interface.go

```diff
@@ -29,11 +30,12 @@ type Driver interface {
 	// count of modules deleted.  Deletion of non-existing prefix will
 	// result in 0, nil being returned.
 	DeleteModules(namePrefix string) (int, error)
```

Comment:
Make sure to:
- [ ] Make "PutModules" and "DeleteModules" package-private since we don't need them any more. We don't want anyone else adding calls to them.
- [ ] Modify the tests for "PutModules" and "DeleteModules" to use "AddTemplate" and "DeleteTemplate" instead.